source/what-is-the-bi-connector.txt

```diff
@@ -31,60 +35,57 @@ The |bi| acts as a layer that translates queries and data between a
 stores no data, and purely serves to bridge your MongoDB cluster with
 business intelligence tools.
 
-.. figure:: /images/bi-connector/components-all.jpg
-   :figwidth: 700px
-   :align: center
-   :alt: Diagram showing that other BI tools communicate with the DSN, which communicates with MongoDB's BI Connector, which in turn communicates with the MongoDB database.
-
 System Components
 -----------------
 
 A complete BI system includes the following components:
 
-MongoDB database
-  Data storage.
+BI Tool
+  Retrieve, visualize, and report on data from a database.
+
+|jdbc| or |odbc| drivers
```

Comment:
Because or is the conjunction used here, I think it should be "driver", singular and not plural. You can use "drivers" with "and".